app/views/requests/_letter_of_consent.html.erb

```diff
@@ -1,5 +1,9 @@
-<%= f.govuk_file_field :letter_of_consent, label: nil, hint: { text: simple_format(t("helpers.hint.request_form.letter_of_consent")) } %>
-<p class="govuk-body-m"><%= t("helpers.hint.request_form.letter_of_consent_file_size") %></p>
+<p class="govuk-body">You must provide a letter of consent from the subject of this request which:</p>
+<%= govuk_list [
+                 "States that they consent to their information being released to you",
+                 "Is signed by them",
+                 "Is dated within the last 6 months"
+               ], type: :bullet %>
 
-<%= f.hidden_field :letter_of_consent_id %>
+<%= f.govuk_file_field :letter_of_consent, label: { size: "s", text: t("request_form.letter_of_consent_title") }, hint: { text: t("helpers.hint.request_form.letter_of_consent_file_size") } %>
```

Comment:
The hint and label text will be picked up automatically if you put it in the expected place in en.yml under helpers.